### ISSUE TITLE
sizeMapping Not Working

### ISSUE THREAD

**Issue by TwisterMc** [jaanauati/react-dfp#244]
I've implemented this library, but the sizeMapping is ignored. On mobile, I'm still getting desktop size ads, and on desktop, sometimes I'll get mobile size ads.

Can anyone verify that sizeMapping works?

I've even tried copying and pasting the example code into mine with the same results.

**Comment by abhishek-parentune** [comment]
same issue

**Comment by abhishek-parentune** [comment]
did you find any alternative?

**Comment by TwisterMc** [comment]
Nope. I had to write my own code for now until we rip out this library completely.

**Comment by abhishek-parentune** [comment]
> Nope. I had to write my own code for now until we rip out this library completely.

if it;s working can you share your librarry?

**Comment by TwisterMc** [comment]
I don't have a library. I have some random code that's custom to the project I was on. It's not sharable unfortunately.
